recipes/install.rb

```diff
@@ -118,6 +118,37 @@
   not_if { node['mongodb']['install_method'] == 'none' }
 end
 
+# Change needed so that updates work properly on debian based systems
+if node['platform_family'] == 'debian'
+  package node['mongodb']['package_name'] + '-server' do
+    options node['mongodb']['packager_options']
+    action :install
+    version package_version
+    not_if { node['mongodb']['install_method'] == 'none' }
+  end
+
+  package node['mongodb']['package_name'] + '-shell' do
+    options node['mongodb']['packager_options']
+    action :install
+    version package_version
+    not_if { node['mongodb']['install_method'] == 'none' }
+  end
+
+  package node['mongodb']['package_name'] + '-tools' do
+    options node['mongodb']['packager_options']
+    action :install
+    version package_version
+    not_if { node['mongodb']['install_method'] == 'none' }
+  end
+
+  package node['mongodb']['package_name'] + '-mongos' do
+    options node['mongodb']['packager_options']
+    action :install
+    version package_version
+    not_if { node['mongodb']['install_method'] == 'none' }
+  end
+end
+
```

Comment:
Having the package manager update all of these at the same time would be preferred.  This should work
```
if node['platform_family'] == 'debian'
  deb_pkgs = %w(
    server
    shell
    tools
    mongos
    ).map { |sfx| "#{node['mongodb']['package_name']}-#{sfx}" }

  package deb_pkgs do
    options node['mongodb']['packager_options']
    action :install
    version package_version
    not_if { node['mongodb']['install_method'] == 'none' }
  end
end
```